lib/workers/repository/extract/index.ts

```diff
@@ -29,14 +30,24 @@ export async function extractAllDependencies(
     }
   };
 
+  const handleCustomManager = (
```

Comment:
move to module scope

Reply:
Could've actually use the old code as is realized after trying to move it to module scope, thanks